tasks/build.py

```diff
@@ -376,7 +382,16 @@ def download_pr(repo_name, branch_name, pr, arch_job_dir):
     # - 'git checkout' base branch of pull request
     # - 'curl' diff for pull request
     # - 'git apply' diff file
-    clone_output, clone_error, clone_exit_code = clone_git_repo(f'https://github.com/{repo_name}', arch_job_dir)
+    log(f"Cloning Git repo via: {clone_via}")
+    if clone_via in (None, 'https'):
+        repo_url = f'https://github.com/{repo_name}'
```

Comment:
nitpick, but github uses `.git` here too

Reply:
Maybe, but the code wasn't doing that before, so I want to stick with what was used before.

But it works both with and without `.git`, so doesn't matter